Packs/CommunityCommonScripts/ReleaseNotes/1_4_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### jq
+
+- Deprecated. Use JMESPath transformer from the FiltersAndTransformers pack instead.
```

Comment:
I would say 'Deprecated. Use JMESPath transformer from the Filters and Transformers content pack instead'  
because that's the name of the pack - https://cortex.marketplace.pan.dev/marketplace/details/FiltersAndTransformers/